aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/CreateHandler.java

```diff
@@ -46,7 +47,15 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(final Amaz
 
         return ProgressEvent.progress(request.getDesiredResourceState(), callbackContext)
                 .then(progress -> setDbClusterParameterGroupNameIfMissing(request, progress))
-                .then(progress -> createDbClusterPGWithTags(proxy, proxyClient, progress, allTags))
+                .then(progress -> Tagging.safeCreate(proxy, proxyClient, this::createDbClusterParameterGroup, progress, allTags))
+                .then(progress -> Commons.execOnce(progress, () -> {
+                            final Tagging.TagSet extraTags = Tagging.TagSet.builder()
+                                    .stackTags(Tagging.translateTagsToSdk(request.getDesiredResourceTags()))
+                                    .resourceTags(new HashSet<>(Translator.translateTagsToSdk(request.getDesiredResourceState().getTags())))
+                                    .build();
```

Comment:
I think I missed this in my commits initially, but we might want to standartize `extraTags` construction. Let's stick to the following:
```java
final Tagging.TagSet extraTags = Tagging.TagSet.builder()
    .stackTags(allTags.getStackTags())
    .resourceTags(allTags.getResourceTags())
    .build();
```
With this construction we can avoid a duplicate type conversion and keep it explicit on what is being included in this structure. I will change it in my earlier commits as well.